stacks/README.md

```diff
@@ -0,0 +1,144 @@
+# Okteto Stacks
+
+A developer-first application format for running containers in Kubernetes
+
+## Why do we need Okteto Stacks?
+
+Helm is a powerful technology to pack your applications and deploy to Kubernetes. It has a powerful engine that makes your application definition declarative. You create new objects or modified them, and Helm takes care of upgrading your resources or deleting the ones you don't need anymore.
+
+But learning Helm can be too complex for the majority of use cases. Developers just want to define the docker image, ports, command, environment variables, volumes and little more. **Stacks** understands a simple docker-compose like format and uses the Helm templating system to traslate this format into Kubernetes resources. Stacks relies on Helm to deploy to Kubernetes, taking advantage of Helm declarative approach and its ecosystem.
```

Comment:
Is it `docker-compose like` or are we supporting `docker-compose` directly? I'd prefeer the second, so that we tell the user to just copy/paste their existing compose.

Reply:
It is not docker-compose. docker-compose has too many formats and particularities that just don't make sense in kubernetes or Okteto.
Docker tried it and the project has been abandoned:
https://github.com/docker/compose-on-kubernetes